src/status_im/chat/models/message.cljs

```diff
@@ -382,16 +387,29 @@
                        :show?            true}
                       chat)))
 
+;; dispatch :update-transactions to update confirmations count
+;; to verify tx initiated with /send command is confirmed
+(defn update-transactions [command-name tx-hash {:keys [with-delay?]} _]
```

Comment:
You asked about this because we update all transactions for every message, right?
If it proves to be a problem, we can try adding an event to update one transaction only.

Reply:
@goranjovic we call `update-transactions` for /send command:
* on sender side once after message was sent, 60 seconds later
* on receiver side twice when message received and 60 secs later.

Reply:
@goranjovic the problem i see is the polling method that we use here. User should wait at least 60 seconds (random parameter for now) to wait for confirmation, while current block time is ~15 seconds. I'd like to provide feedback faster.

Second issue, what if 60 seconds later there are no confirmations - we don't schedule new `update-transactions` event, we just do nothing. Of course user could go to wallet which will run update (or restart the app), but it doesn't make good UX.

Reply:
Ah, ok I see. Let me think